electron/constants.ts

```diff
@@ -43,8 +79,13 @@ function getGOGdlBin() {
         isWindows ? 'gogdl.exe' : 'gogdl'
       )
   )
+
+  // dont fix windows issues
+  if (bin.includes(' ')) {
+    return `"${bin}"`
```

Comment:
Based on that, we should make sure gogdl is not wrapped twice in quotes when actually executing it
`gog/games.ts`, `gog/library.ts`

Reply:
so I will return to the old code then. Since GOG is working normally on 2.2.4